pom.xml

```diff
@@ -3,9 +3,9 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<parent>
-		<groupId>org.scijava</groupId>
-		<artifactId>pom-scijava</artifactId>
-		<version>12.0.0</version>
+		<groupId>net.imagej</groupId>
+		<artifactId>pom-imagej</artifactId>
+		<version>16.5.0</version>
```

Comment:
Please fix this... it needs to be `org.scijava:pom-scijava:12.0.0`, _not_ the obsolete `pom-imagej` anymore.